### ISSUE TITLE
Staging Review finding: Heading structure doesn't match similar content in other VA.gov products

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#87890]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** VA BTSSS - Travel Pay
**Product:** Travel Pay
**Feature:** MVP

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that doesn't match other content on VA.gov.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-is-similar-to-other-content-on-vagov)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

There are a number of places on VA.gov that follow the same general page structure here: search/sort/filtering tools, then a "Showing X - Y of Z" message, then a list of results.

In most (though not quite all), the "Showing X - Y of Z" message is coded as an H2 heading even when styled to look like regular text. As currently coded, this page presents that message as a `p` paragraph.

Many screen reader users navigate page content via heading, eg. skipping from heading to heading to find the section of content they're looking for. Making the "Showing..." message a heading gives them a consistent start point for the list of results.

This page is straightforward enough (and the cards clearly indicated with headings themselves) that the actual impact of this page being inconsistent probably isn't huge. But it's a content structure that's common enough on VA.gov that I would expect many screen reader users to be looking for it.

## Recommended action

Recommend coding the "Showing..." message as an H2 and styling as regular text. Note that you'll need to adjust the heading levels for the headings that come later in the page to properly nest them under this new H2 (eg. each card heading becomes an H3, "Where" becomes an H4, "Claim Details" becomes an H4).

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
